plugins/amazonq/codewhisperer/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codewhisperer/popup/CodeWhispererPopupManager.kt

```diff
@@ -342,11 +346,6 @@ class CodeWhispererPopupManager {
                 popup.size = popup.preferredContentSize
             }
         } else {
-            val originalAutoPopupCompletionLookup = CodeInsightSettings.getInstance().AUTO_POPUP_COMPLETION_LOOKUP
-            CodeInsightSettings.getInstance().AUTO_POPUP_COMPLETION_LOOKUP = false
-            Disposer.register(popup) {
-                CodeInsightSettings.getInstance().AUTO_POPUP_COMPLETION_LOOKUP = originalAutoPopupCompletionLookup
-            }
```

Comment:
duplicate of https://github.com/aws/aws-toolkit-jetbrains/pull/4582/files#diff-4ee308f7dbc11eba350ccc2886bc9cc5a6a3a8590de5b3e7d10a43c18d7bcb27L698

and moved to CodeInsightSettingFacade